sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/README.md

```diff
@@ -77,22 +81,21 @@ This indicates that the Event Hubs service did not respond to an operation withi
 #### Message Size Exceeded
 
 Event data, both individual and in batches, have a maximum size allowed.  This includes the data of the event, as well as any associated metadata and system overhead.  The best approach for resolving this error is to reduce the number of events being sent in a batch or the size of data included in the message.  Because size limits are subject to change, please refer to [Azure Event Hubs quotas and limits](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-quotas) for specifics.  
-
 ### Other exceptions
 
 For detailed information about these and other exceptions that may occur, please refer to [Event Hubs messaging exceptions](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-messaging-exceptions). 
 
 ## Next steps
 
-Beyond the scenarios discussed, the Azure Event Hubs blob storage checkpoint library offers support for additional scenarios to help take advantage of the full feature set of the Event Processor.  In order to help explore some of these scenarios, the Azure Event Hubs blob storage checkpoint library offers a [project of samples](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/samples) to serve as an illustration for common scenarios.
+Beyond the scenarios discussed, the Azure Event Hubs blob storage checkpoint library offers support for additional scenarios to help take advantage of the full feature set of the `EventProcessor`.  In order to help explore some of these scenarios, the Azure Event Hubs blob storage checkpoint library offers a [project of samples](./samples) to serve as an illustration for common scenarios.
```

Comment:
This will be a follow-up set of changes, once there is actually code to inspect and extract from.  The intent of this PR is to fill in content where possible in the absence of a completed library to draw from.